pkg/operator/get_operator_split.go

```diff
@@ -0,0 +1,108 @@
+package operator
+
+import (
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common/flags"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/operator/split"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/rewards"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/telemetry"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/utils"
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	eigenSdkUtils "github.com/Layr-Labs/eigensdk-go/utils"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/urfave/cli/v2"
+)
+
+func GetOperatorSplitCmd(p utils.Prompter) *cli.Command {
+	var operatorSplitCmd = &cli.Command{
+		Name:  "get-rewards-split",
+		Usage: "Get operator rewards split",
+		Action: func(cCtx *cli.Context) error {
+			return GetOperatorSplit(cCtx)
+		},
+		After: telemetry.AfterRunAction(),
+		Flags: getOperatorSplitFlags(),
```

Comment:
this is still getting the set flags which has signer. this should have it's own flags without signer